experimental/generation/generator/test/commands/dialog/forms/unittests1.schema

```diff
@@ -0,0 +1,117 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+        "Quantity": {
+            "type": "number",
+            "minimum": 1,
+            "maximum": 10,
+            "$entities": [
+                "number"
+            ]
```

Comment:
You should not need $entities for anything where the type or format directly defines the entities.  In this case this should come in through the number.lg template.